desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -298,6 +298,13 @@ def explain(self, notebook, snippet):
     if statement:
       if self.options['url'].startswith('bigquery://'):
         explanation = ''
+      else:
+        result = connection.execute('EXPLAIN '+ statement)
+  
+        explanation = [dict(r) for r in result]
+        explanation = "\n".join("{!r}: {!r},".format(k, v) for dictt in explanation for k, v in dictt.items())
```

Comment:
Why !r nd not simpler?


https://docs.python.org/3/tutorial/inputoutput.html#the-string-format-method